qcore/test/test_source_site_dist/test_src_site_dist_calc.py

```diff
@@ -0,0 +1,115 @@
+import os
+import pickle
+
+import numpy as np
+import pytest
+
+from qcore import geo, src_site_dist as ssd
+
+INPUT = "input"
+OUTPUT = "output"
+
+
+def test_calc_rrub_rjb(set_up):
```

Comment:
The issues your having from the tests stem from this set_up which was a shared function for the tests in IM_calculation which setup the testing env with input output I believe